doc/source/whatsnew/v1.2.0.rst

```diff
@@ -495,6 +495,7 @@ Groupby/resample/rolling
 - Bug in :meth:`DataFrame.groupby.rolling` returning wrong values with partial centered window (:issue:`36040`).
 - Bug in :meth:`DataFrameGroupBy.rolling` returned wrong values with timeaware window containing ``NaN``. Raises ``ValueError`` because windows are not monotonic now (:issue:`34617`)
 - Bug in :meth:`Rolling.__iter__` where a ``ValueError`` was not raised when ``min_periods`` was larger than ``window`` (:issue:`37156`)
+- Bug in :meth:`df.groupby(..).quantile() <pandas.core.groupby.DataFrameGroupBy.quantile>` and :meth:`df.resample(..).quantile() <pandas.core.resample.Resampler.quantile>` raised ``TypeError`` when values to calculate quantile over where ``Timedelta`` (:issue:`29485`)
```

Comment:
where -> were. Also, I think it's okay to just say e.g. "when values were of type ``Timedelta``"

Reply:
Done